src/ansys/fluent/core/utils/fluent_version.py

```diff
@@ -20,3 +37,97 @@ def get_version_for_file_name(version: Optional[str] = None, session=None):
         version = get_version(session)
 
     return "".join(version.split(".")[0:2])
+
+
+class FluentVersion(Enum):
```

Comment:
@ansaboutin we want to have this class in a separate module so that we can import it cleanly from flobject.py without dragging in other unwanted dependencies that are unneeded in the Fluent context.